### ISSUE TITLE
IcedCoffeeScript and CoffeeScript bundles exclude one another

### ISSUE THREAD

**Issue by sheldonth** [dhoelzgen/iced-coffee-script-tmbundle#6]
If I cd /Applications/TextMate.app/Contents/SharedSupport/Bundles

and double click the IcedCoffeScriptBundble.tmbundle then my CoffeeScript bundle is no longer shown in the bundle selection drop down in TextMate2. If I then re-install CoffeeScript the IcedCoffeeScript vanishes and so forth.

**Comment by dhoelzgen** [comment]
Yep, this is just a fork of the CoffeeScript bundle, with additions for the changed syntax and the other command. Since CoffeeScript is (or, at the time of writing the plugin, was) a subset of iced coffee script, I didn't see the need for having them run in parallel.

However, I neither use TM or SublimeText atm, and I haven't used IcedCoffeeScript for a while, just having the repo up for people still using this bundle, so if you like to change that, feel free to send me a pull request.
